src/common/ContractsBase.ts

```diff
@@ -3,7 +3,7 @@ import BN from 'bn.js'
 import ChildERC20Artifact from 'matic-protocol/contracts-core/artifacts/ChildERC20.json'
 import ChildERC721Artifact from 'matic-protocol/contracts-core/artifacts/ChildERC721.json'
 import ChildMaticArtifact from 'matic-protocol/contracts-core/artifacts/MaticChildERC20.json'
-
+import transferWithSigABI from './transferWithSigABI.json'
```

Comment:
No need to add this, You can access `transferWithSig` from `ChildERC721Artifact `

Reply:
worked, thanks!